openbb_sdk/sdk/provider/openbb_provider/registry_map.py

```diff
@@ -87,8 +90,19 @@ def _get_map(self, registry: Registry) -> Tuple[MapType, MapType]:
                     "QueryParams": extra_query,
                     "Data": extra_data,
                 }
+
+                in_return_map = return_map.get(model_name, return_type)
+                if union_return_map.get(model_name, None) is None and (
+                    get_origin(return_type) != list
+                    and get_origin(in_return_map) == list
```

Comment:
@tehcoderer  here can't we just check if `get_origin(return_type) != get_origin(in_return_map)` so it works for any type besides `list`?